### ISSUE TITLE
Time filter "sticks", but can't be reset

### ISSUE THREAD

**Issue by pbb72** [mapillary/mapillary_issues#1438]
After setting a time filter on the map page, this filter stays active when switching to a photo page. This is nice, because only the filtered sequences show up in the mini-map. However, when switching back to the map page, with the filter still active, there is no option to reset the time filter.

To get back to all sequences, you first have to define a new time filter, only after that can you reset the filter.

**Comment by de-vries** [comment]
duplicate of #1251

**Comment by pbb72** [comment]
Correct, it's a duplicate
